aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/AWSCognitoAuthPlugin.java

```diff
@@ -375,14 +375,56 @@ public void onError(Exception error) {
 
     @Override
     public void signIn(
-            @Nullable String username,
-            @Nullable String password,
-            @NonNull final Consumer<AuthSignInResult> onSuccess,
-            @NonNull final Consumer<AuthException> onException
+        @Nullable String username,
```

Comment:
Could you revert this indentation?   As a convention throughout the codebase, we generally indent 8 spaces, when method signatures are wrapped, to differentiate it from the method implementation, which is indented 4 spaces.